subprojects/docs/src/docs/design/gradle-module-metadata-latest-specification.md

```diff
@@ -6,7 +6,7 @@ Gradle publishes Gradle Module Metadata along with traditional metadata. Gradle
 
 ## Goal
 
-This document describes version 1.0 of the Gradle module metadata file. A module metadata file describes the contents of a _module_, which is the unit of publication for a particular repository format, such as a module in a Maven repository. This is often called a "package" in many repository formats.
+This document describes version 1.1 of the Gradle module metadata file format. A module metadata file describes the contents of a _module_, which is the unit of publication for a particular repository format, such as a module in a Maven repository. This is often called a "package" in many repository formats.
```

Comment:
:x: let's create a new version of this document, so that in a single directory we can find all revisions of the format. This should help 3rd party when trying to understand the different versions of the GMM format.

Reply:
This was already done and this is the 1.1 (latest) copy. We just forgot to update this one mention of 1.0.

See: #10230

Reply:
Ah good, I totally missed this, great!